svgnative/CMakeLists.txt

```diff
@@ -95,7 +128,7 @@ endif()
 if(EXISTS "${CMAKE_TOOLCHAIN_FILE}")
     # do not apply any global settings if the toolchain
     # is being configured externally
-elseif(MSVC)
+elseif(WIN32)
```

Comment:
I am not sure if that is correct. The following settings are specific to VS. Have you tried building with VS+clang or migw?

Reply:
OK, I would revert them to MSVC (I was thinking about Clang, but overlooked about MinGW). I already confirmed clang-cl.exe toolset (provided as a component of MSVS) can build the set, by changing the toolset from MSVS GUI. It seems that changing the toolset on Windows commandline is quite complicated (please find [How to use clang-cl and lld-link with Visual Studio generator](https://gitlab.kitware.com/cmake/cmake/issues/19174)). I would postpone it as future issue.